amazon-ecr-credential-helper.spec

```diff
@@ -114,6 +112,11 @@ install -D -m 0644 \
 rm -rf %{buildroot}
 
 %changelog
+* Tue Jan 7 2020 Samuel Karp <skarp@amazon.com> - 0.4.0-1
+- Added support for chaining assumed roles in the shared config file
+- Added support for Web Identities and IAM Roles for Service Accounts (IRSA)
```

Comment:
I checked Fedora's Packaging Guidelines and also ran `rpmlint` on this spec after we chatted (offline) about the long, newline broken changelog entry - both have nothing to say at all on the matter, so ship it!